$OpenBSD$
Allow libkipi to be found even without pkg-config file.
--- cmake/modules/FindKipi.cmake.orig	Thu Aug 15 00:35:09 2013
+++ cmake/modules/FindKipi.cmake	Thu Aug 15 00:36:56 2013
@@ -81,7 +81,7 @@ ELSE(KIPI_INCLUDE_DIR AND KIPI_LIBRARIES AND KIPI_DEFI
       PKG_CHECK_MODULES(KIPI libkipi>=${Kipi_FIND_VERSION})
     ENDIF(NOT WIN32)
 
-    FIND_LIBRARY(KIPI_LIBRARIES NAMES libkipi PATHS ${KIPI_LIBRARY_DIRS} ${LIB_INSTALL_DIR} ${KDE4_LIB_DIR})
+    FIND_LIBRARY(KIPI_LIBRARIES NAMES kipi PATHS ${KIPI_LIBRARY_DIRS} ${LIB_INSTALL_DIR} ${KDE4_LIB_DIR})
     FIND_PATH(KIPI_INCLUDE_DIR NAMES libkipi/version.h PATHS ${KIPI_INCLUDE_DIRS} ${INCLUDE_INSTALL_DIR} ${KDE4_INCLUDE_DIR})
     SET(KIPI_VERSION_H_FILENAME "${KIPI_INCLUDE_DIR}/libkipi/version.h")
     SET(KIPI_DEFINITIONS ${KIPI_CFLAGS} CACHE STRING "Kipi definitions")
